fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostCalculator.java

```diff
@@ -133,5 +134,32 @@ public CostEstimate visitPhysicalHashJoin(PhysicalHashJoin<Plan, Plan> physicalH
                     0);
         }
 
+        @Override
+        public CostEstimate visitPhysicalNestedLoopJoin(PhysicalNestedLoopJoin<Plan, Plan> nestedLoopJoin,
```

Comment:
Similar to `ChildOutputPropertyDeriver`,  maybe we should revisit this logic when tuning join logic in cascades.

Reply:
yes, so i add a TODO in this function